lib/richtext_controller.dart

```diff
@@ -216,12 +216,31 @@ class RichTextController {
       applyTextColor(),
       applyBackgroundTextColor(),
     ]);
+    onFocus?.call();
     showRichTextView();
   }
 
-  void onCreateHTMLEditor(HtmlEditorApi editorApi) {
+  void onCreateHTMLEditor(
+    HtmlEditorApi editorApi, {
+    ScrollController? scrollController,
+    VoidCallback? onFocus,
+    VoidCallback? onEnterKeyDown,
+  }) {
     htmlEditorApi = editorApi;
-    editorApi.onFocus = editorOnFocus;
+    editorApi.onFocus = () {
+      editorOnFocus.call(onFocus);
+    };
+
+    editorApi.onKeyDown = () {
+      scrollController?.animateTo(
```

Comment:
IMO, you should use `debounce_time` in case the user repeatedly presses the `enter` key or the `newline` character, this scrolling will throw an exception. Let's test this case again.

Reply:
@dab246 No need for it. I tested

https://user-images.githubusercontent.com/107173849/193966608-17a3d3a0-cfaa-40aa-b683-d89714498215.mov